### ISSUE TITLE
showInactive(); not working as before after windows 10 Anniversary update    

### ISSUE THREAD

**Issue by molster** [electron/electron#6908]
**TL;DR - showInactive() no longer working as expected on windows with with Anniversary update, works correct still on all other versions of windows, including 10 without the update**

-version=0.33.9 

Hello, We have a window that is setup as follows

```
overlayWindow = new BrowserWindow({
    width: 200,
    height: 85,
    frame: false,
    resizable: false,
    'auto-hide-menu-bar': true,
    'use-content-size': false,
    icon: __dirname + '/icon.png',
    show: false,
    transparent: aeroEnabled,
    title: "Overlay",
    'always-on-top': true,
    'skip-taskbar': true,
});
```

The window is hidden by default,  The window is shown via the following call.

```
function showDisplay() {
    window.showInactive();
    window.setAlwaysOnTop(true);
    displayOn = true;
}
```

The issue here is, on windows 7 - 10 (without the anniversary update) everything works as expected. However, users who have windows 10 with the update, have the task bar pop up when the overlay window is shown.  

Something is now triggering the task bar.

Note: we have found that we need to call AlwaysOnTop again, as it had mixed results if we only set it at window creation, removing this from the showDisplay function has no impact on showInactive working on windows 10 Annv or not

**Comment by zcbenz** [comment]
😞 I'm not sure if we can do anything on this since it seems to be a bug of Anniversary update.

**Comment by molster** [comment]
Yeah, was hoping maybe there was just some API change with the update that can be accounted for ..ect

Open to any ideas.

**Comment by dimon222** [comment]
I think latest Windows Update resolved the issue

**Comment by molster** [comment]
This would be really good news. I am currently out of town on business, when im back, ill attempt to verify on my end as well and close issue if needed, or reply if till an issue

**Comment by molster** [comment]
Had one of my devs update to latest windows and give it a go. Still having the new effect of showing the taskbar.

**Comment by kevinsawicki** [comment]
@molster are you still seeing this happen with newer Electron versions and Windows 10 updates? Thanks.

**Comment by molster** [comment]
Sorry for the Delay, been busy with the holidays and other fun things happening.

we are not on the latest Electron due to needing a rather large re-write to get to it.  If it appears there is a possibility that a fix has been included, we will look into re-write some code to test it out. Ill write back findings on that by end of next week.

**Comment by molster** [comment]
We have tried with latest electron, And the issue is still present. 

Works great on all versions of windows up to those using anniversary update on windows 10

**Comment by kevinsawicki** [comment]
@molster would you be able to try Electron 1.6 and see if you are still experiencing it?

**Comment by molster** [comment]
@kevinsawicki no luck. looks like 1.6.0 has the same result (on windows 10 with anniversary update. it will pop-up the task bar)

Is there any other info I can provide to help? would be happy to include a video of it on windows 8.1 vs 10 (I no longer have any computers without the anniversary update, or id show how it changed with the windows update for 10)

**Comment by kevinsawicki** [comment]
> Is there any other info I can provide to help? would be happy to include a video of it on windows 8.1 vs 10 (I no longer have any computers without the anniversary update, or id show how it changed with the windows update for 10)

Yeah, a quick video would be helpful, thanks for offering 👍 🎥

**Comment by zcbenz** [comment]
Closing until more information is provided.
